File: exercism/python/raindrops/raindrops.py
#!/usr/bin/python3
# https://exercism.io/my/solutions/e3c1cfe830fe4c3498baca47408220c3


def raindrops(number):
    factors = [3, 5, 7]
    strings = ['Pling', 'Plang', 'Plong']
    outputs = [w if number % f == 0 else '' for f, w in zip(factors, strings)]
    return ''.join(outputs) or str(number)

File: exercism/go/raindrops/raindrops.go
// https://exercism.io/my/solutions/741b0cefdb5141eeaf2d81ca70adf131

package raindrops

import "strconv"

// Convert returns funny strings
func Convert(number int) string {
	type Raindrop struct {
		factor   int
		response string
	}

	raindrops := []Raindrop{
		{3, "Pling"},
		{5, "Plang"},
		{7, "Plong"},
	}

	output := ""
	for _, r := range raindrops {
		if number%r.factor == 0 {
			output += r.response
		}
	}

	if output == "" {
		return strconv.Itoa(number)
	}
	return output
}
